### PR TITLE
Add recipe for Browserify

### PR DESCRIPTION
**Pull request by erdostom**
My solution for including modules in the app.

Love this project, thank you for your work!

### CODE REVIEW

File: docs/recipes/browserify.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+### 2. Edit your `scripts` task
+
+```diff
+gulp.task('scripts', () => {
+-  return gulp.src('app/scripts/**/*.js')
++
++  const b = browserify({
++    entries: 'app/scripts/main.js',
++    transform: babelify,
```
Comment (kevva) [inline docs/recipes/browserify.md:32]: Should probably remove this in favor of `b.transform('babelify')`.

**Comment by erdostom** [comment]
Why?

**Comment by kevva** [comment]
I guess you meant to answer my comment. Because just using `babelify` as a `transform` option does nothing. How would you configure `babelify` inside the `transform` option?

**Comment by erdostom** [comment]
Yes, sorry. I don't have any config happening there, just using the existing config in the `.babelrc`.

**Comment by kevva** [comment]
Oh, yea. That should work then :). Just discard my comment.

File: docs/recipes/browserify.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+```js
+module.exports =  {
+  speak: () => console.log('hello world')
+}
+```
+
+And then in your `main.js`:
+
+```js
+let foo = require('./foo')
```
Comment (kevva) [inline docs/recipes/browserify.md]: `const foo = require('./foo');`

File: docs/recipes/browserify.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+## Usage
+
+- Put your modules in `app/scripts`, and require them in your `main.js`.
+
+For example, you could have `foo.js`:
+
+```js
+module.exports =  {
+  speak: () => console.log('hello world')
+}
```
Comment (kevva) [inline docs/recipes/browserify.md]: `module.exports.speak = () => console.log('hello world');`

**Comment by erdostom** [comment]
thanks @kevva, will make those changes. do you think it would be worthwhile to add some kind of module support to the core generator? what is the criteria for deciding if a feature is a recipe / core change? thanks again

**Comment by erdostom** [comment]
just checking if you guys need anything else to get this PR merged

**Comment by kevva** [comment]
Nope, this looks great :+1:

**Pull request merged**